src/World1/world1/plots.jl

```diff
@@ -0,0 +1,359 @@
+
+    plotvariables(sol, (t, 1900, 2100), _variables_2(); title="W1-7/5-2", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_3(; kwargs...)
+    new_params = copy(_params)
+    new_params[:cigc] = 0.5
+
+    new_inits = copy(_inits)
+    new_inits[:ciaf] = 0.25
+
+    sol = solve(standard_run(inits=new_inits, params=new_params), (1900, 2400))
+
+    plotvariables(sol, (t, 1900, 2400), _variables_3(); title="W1-7/5-3", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_4(; kwargs...)
+    new_params = copy(_params)
+    new_params[:brn] = 0.025
+
+    sol = solve(standard_run(params=new_params), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_4(); title="W1-7/5-4", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_5(; kwargs...)
+    new_params = copy(_params)
+    new_params[:brn] = 0.035
+
+    new_inits = copy(_inits)
+    new_inits[:ciaf] = 0.3
+
+    sol = solve(standard_run(params=new_params), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_5(); title="W1-7/5-5", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_6(; kwargs...)
+    new_params = copy(_params)
+    new_params[:brn] = 0.032
+
+    new_inits = copy(_inits)
+    new_inits[:ciaf] = 0.3
+
+    sol = solve(standard_run(params=new_params), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2000), _variables_6(); title="W1-7/5-6", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_7(; kwargs...)
+    new_params = copy(_params)
+    new_params[:pols] = 10e9
+
+    sol = solve(standard_run(params=new_params), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_7(); title="W1-7/5-7", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_8(; kwargs...)
+    new_params = copy(_params)
+    new_params[:pdn] = 26.5
+
+    sol = solve(standard_run(params=new_params), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_8(); title="W1-7/5-8", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_9(; kwargs...)
+    new_params = copy(_params)
+    new_params[:pdn] = 26.5
+
+    new_tables = copy(_tables)
+    new_tables[:cipc] = (0.02, 0.05, 0.1, 0.14, 0.16, 0.18)
+
+    sol = solve(standard_run(params=new_params, tables=new_tables), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_9(); title="W1-7/5-9", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_10(; kwargs...)
+    new_params = copy(_params)
+    new_params[:pdn] = 26.5
+
+    new_tables = copy(_tables)
+    new_tables[:cipc] = (0.02, 0.05, 0.1, 0.14, 0.16, 0.18)
+    new_tables[:polat] = (0.5, 1.0, 1.6, 2.4, 3.2, 4.0, 4.8)
+
+    sol = solve(standard_run(params=new_params, tables=new_tables), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_10(); title="W1-7/5-10", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_11(; kwargs...)
```

Comment:
The generated plot is not included in the book, but the modifications are as described on page 55.